server/models/account/user.ts

```diff
@@ -637,6 +695,14 @@ export class UserModel extends Model<UserModel> {
   toFormattedJSON (this: MUserFormattable, parameters: { withAdminFlags?: boolean } = {}): User {
     const videoQuotaUsed = this.get('videoQuotaUsed')
     const videoQuotaUsedDaily = this.get('videoQuotaUsedDaily')
+    const videoCount = this.get('videoCount')
+    const videoAbuseCount = this.get('videoAbuseCount') as string
+    const videoAbuseCreatedCount = this.get('videoAbuseCreatedCount')
+    const videoCommentCount = this.get('videoCommentCount')
+
+    function parseAbuseCount (videoAbuseCount: string, position: number) {
```

Comment:
I'm not sure to understand what is `videoAbuseCount` or `position`. Try to build your data directly using postgresql instead

Reply:
I'm just reusing the `INNER JOIN` and making `COUNT` with different conditions, so the literal output is in the form `(x,y)` (a string) and this function extracts the stat at position z. for instance, for `z = 0`, we get `x`. For `z = 1`, we get `y`. But that is only worth the trouble if making another join from the ground up is costly. It is, right? Right? :sweat_smile: I have no idea how/if sequelize optimises literals.

Reply:
I think you can just do multiple `COUNT` with their own `FILTER` that are mapped to a specific name. I don't think it will change a lot the query performance